presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -333,6 +326,11 @@ private ShardInfo createShardInfo(UUID shardUuid, File file, Set<String> nodes,
             return shardDelta(shardUuid, Optional.empty());
         }
 
+        if (isBackupAvailable()) {
```

Comment:
This check seem unnecessary since the backup manager can be used even if backups are disabled